spec/index.html

```diff
@@ -492,9 +492,10 @@ <h3>IRIs</h3>
       of [[!RFC3987]]. Further normalization MUST NOT be performed when
       comparing IRIs for equality.</p>
 
-      <p class="issue" data-number="15">
-        The group is aware that work has not progressed on [[RFC3987]],
-        and is considering alternatives.</p>
+      <p class="issue" data-number="4">
+        [[RFC3987]] never made it past the Proposed Standard phase
```

Comment:
We could take this out.

As [pointed out](https://github.com/w3c/rdf-concepts/issues/15#issuecomment-1436613178), many RFC are "Proposed Standard" (including URNs).

Reply:
Yes. I noted this removal option in another review of the same basic text in another copy of this shared-yet-not-shared document/content. (I have generally been focused on editing the text in place, rather than the larger meaning of that text.)

Perhaps we need a distinct repo for the shared/common content, from which all that shared content is imported by whatever means. I know that one experiment failed because it broke previews. I think there's another experiment in progress. I hope that or another like it bears fruit soon!

Reply:
Removed.